app/src/main/java/com/android/example/FakeContentProviderFilenameSanitization.kt

```diff
@@ -0,0 +1,68 @@
+package com.android.example
+
+import android.database.Cursor
+import android.provider.OpenableColumns
+import java.io.File
+
+
+class FakeContentProviderFilenameSanitization {
+    private fun trustProvidedFilename_shouldWarn(cursor: Cursor) {
+        val id = cursor.getColumnIndex(OpenableColumns.DISPLAY_NAME)
+        val fileName = cursor.getString(id)
+
+        val fileObject = File("./", fileName)
+    }
+
+    private fun cursorFilenameAssignedToVariables(cursor: Cursor) {
+        val fileName = cursor.getString(cursor.getColumnIndex(OpenableColumns.DISPLAY_NAME))
+
+        val fileObject = File("./", fileName)
+    }
+
+    private fun additionalPerformedOperations(cursor: Cursor) {
+        var temp = 1 + 2
+
+        val id = cursor.getColumnIndex(OpenableColumns.DISPLAY_NAME)
+        println(temp)
+        val fileName = cursor.getString(id)
+
+        temp += 2
+        val fileObject = File("./", fileName)
+    }
+
+    private fun callChainStartingAtContext(cursor: Cursor) {
```

Comment:
This method seems to be a duplicate of  `trustProvidedFilename_shouldWarn`.

Reply:
Thanks, I deleted trustProvidedFilename_shouldWarn! I had copied your first two test cases into FakeContentProviderFilenameSanitization as it was known code to trigger the lint firing, but didn't check that they were duplicates. Oops :)